scripts/test-cli-integration

```diff
@@ -0,0 +1,190 @@
+#!/bin/bash
+
+set -eou pipefail
+set -x
+
+cd "$(dirname "$0")/.."
+
+key_a=$(prio keygen)
+key_b=$(prio keygen)
+shared=$(prio shared-seed)
+
+###########################################################
+# encode-shares
+###########################################################
+
+mkdir -p working/client
+
+cat << EOF > working/client/data.ndjson
+[1, 0, 0]
+[1, 1, 0]
+[1, 1, 1]
+EOF
+
+jq -c '.' working/client/data.ndjson
+
+python -m prio encode-shares \
+    --n-data 3 \
+    --batch-id test \
+    --public-key-internal $(jq -r ".public_key" <<< $key_a) \
+    --public-key-external $(jq -r ".public_key" <<< $key_b) \
+    --input working/client/data.ndjson \
+    --output-A working/server_a/raw/ \
+    --output-B working/server_b/raw/
+
+jq -c '.' working/server_a/raw/data.ndjson
```

Comment:
Could we make things like `working/server_a/raw/` bash variables like:

```bash
SERVER_A_WORKDIR=working/server_a/raw/
```

This makes it a little easier to catch typos and errors.

Reply:
I refactored this to extract the environment variables out, it looks a lot nicer now.